NEWS.md

```diff
@@ -1,8 +1,28 @@
+# admiralext 0.0.2
+
+## New Features
+
+ - New `renv` lock files were setup (#48)
+
+## Updates of Existing Functions
+
```

Comment:
oohhh could you put in the stuff about the details you updated in the `my_first_fcn()`.  I think being able to see this is nice.

Reply:
ah...i see it in next section and this isn't an update to the function...scratch it